apps/openssl-vms.cnf

```diff
@@ -60,6 +60,13 @@ default = default_sect
 # included fipsmodule.cnf.
 # fips = fips_sect
 
+# If no providers are activated explicitly, the default one is activated implicitly.
+# See man 7 OSSL_PROVIDER-default for more details.
+#
+# If you add a section explicitly activating any other provider(s),
+# you most probably need to explicitly activate the default provider,
+# otherwise it becomes unavailable in openssl.
+# As a consequence the system may for example become unreachable via SSH.
```

Comment:
I think the reference to SSH is confusing. I'd drop this line.

Reply:
@mattcaswell, but that's exactly the problem I want to avoid. For a not-deeply-involved person breaking the SSH access via the editing the openssl config is an unpredictable consequence

Reply:
In that case I think you should replace this with a much more broadly worded warning. OpenSSL does not do SSH. If you happen to have an SSH application installed that uses OpenSSL then it could impact your SSH access but this is very indirect.

Reply:
I'd accept any wording that will save users from such a situation

Reply:
How about this compromise wording that drops the explicit reference to SSH:
"As a consequence applications depending on OpenSSL may not work correctly which could lead to significant system problems including inability to remotely access the system."

Reply:
Pushed, thanks!